content/en/docs/ambient/install/istioctl-installation/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Install with Istioctl
```

Comment:
Consider using "Istio CLI" (instead of istioctl). I am thinking about new users who might not know what istioctl is.